packages/database/src/migrations/20221005045601-MoveFetchIntoTransformInReports-modifies-data.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const convertToNewConfig = config => {
+  const { fetch, transform, ...restOfConfig } = config;
+  if (!fetch) {
+    return config;
```

Comment:
Some reports are no 'customReports' that don't have a `fetch` or `transform` step, so just ignore those